src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,215 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// <returns>The navigation animation that occurs when a page is navigated.</returns>
 		public static FlyoutBehavior GetFlyoutBehavior(BindableObject obj) => (FlyoutBehavior)obj.GetValue(FlyoutBehaviorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetFlyoutBehavior']/Docs/*" />
+
+		/// <summary>
+		/// Sets the behavior used to open the flyout.
+		/// </summary>
+		/// <remarks>
+		/// The flyout can be accessed through the hamburger icon or by swiping from the side of the screen.
+		/// However, this behavior can be changed by setting the <see cref = "FlyoutBehavior" /> attached property.
+		/// </remarks>
+		/// <param name="obj">The object that modifies the Shell behavior used to open the flyout.</param>
+		/// <param name="value">The behavior used to open the flyout.</param>
 		public static void SetFlyoutBehavior(BindableObject obj, FlyoutBehavior value) => obj.SetValue(FlyoutBehaviorProperty, value);
 
+		/// <summary>
+		/// Gets the width of the flyout.
+		/// </summary>
+		/// <param name="obj">The object that modifies the width of the flyout.</param>
+		/// <returns>The width of the flyout.</returns>
 		public static double GetFlyoutWidth(BindableObject obj) => (double)obj.GetValue(FlyoutWidthProperty);
+
+		/// <summary>
+		/// Sets the width of the flyout.
+		/// This enables scenarios such as expanding the flyout across the entire screen.
+		/// </summary>
+		/// <param name="obj">The object that modifies the width of the flyout.</param>
+		/// <param name="value">Defines the width of the flyout.</param>
 		public static void SetFlyoutWidth(BindableObject obj, double value) => obj.SetValue(FlyoutWidthProperty, value);
 
+		/// <summary>
+		/// Gets the height of the flyout.
+		/// </summary>
+		/// <param name="obj">The object that modifies the height of the flyout.</param>
+		/// <returns>The height of the flyout.</returns>
 		public static double GetFlyoutHeight(BindableObject obj) => (double)obj.GetValue(FlyoutHeightProperty);
-		public static void SetFlyoutHeight(BindableObject obj, double value) => obj.SetValue(FlyoutHeightProperty, value);
 
+		/// <summary>
+		/// Sets the height of the flyout.
+		/// </summary>
+		/// <remarks>
+		/// The height of the flyout can be customized by setting the Shell.FlyoutHeight attached properties to double value.
+		/// This enables scenarios such as reducing the height of the flyout so that it doesn't obscure the tab bar.
+		/// </remarks>
+		/// <param name="obj">The object that modifies the height of the flyout.</param>
+		/// <param name="value">Defines the height of the flyout.</param>
+		public static void SetFlyoutHeight(BindableObject obj, double value) => obj.SetValue(FlyoutHeightProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarIsVisible']/Docs/*" />
+		/// <summary>
+		/// Gets if the navigation bar is visible when a page is presented. 
```

Comment:
```suggestion
		/// Gets a value indicating if the navigation bar is visible when when the given <paramref name="obj"/> is active. 
```

I would also add a remark that obj should be a page